src/main/java/com/github/jreddit/parser/entity/imaginary/CommentTreeElement.java

```diff
@@ -0,0 +1,6 @@
+package com.github.jreddit.parser.entity.imaginary;
+
+
+public interface CommentTreeElement {
```

Comment:
Reason added. 

Interface to abstract over the two possible elements in a comment tree, namely a {@link Comment} or a {@link More} thing. If an object is of the type of this interface, it means that it <i>must</i> be either an {@link Comment} or a {@link More} thing.